application/wlp-web-java-spring/src/main/java/webapp/tier/db/MysqlService.java

```diff
@@ -0,0 +1,125 @@
+package webapp.tier.db;
+
+import java.security.NoSuchAlgorithmException;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import javax.sql.DataSource;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import webapp.tier.util.CreateId;
+import webapp.tier.util.GetConfig;
+
+public class MysqlService {
+
+	private static Logger logger = LoggerFactory.getLogger(MysqlService.class);
+	private static String sqlkey = GetConfig.getResourceBundle("mysql.insert.msg.id");
+	private static String sqlbody = GetConfig.getResourceBundle("mysql.insert.msg.body");
+	private static String message = GetConfig.getResourceBundle("common.message");
+	private static String jndiname = GetConfig.getResourceBundle("mysql.jndi.name");
+	private static String sqlselect = GetConfig.getResourceBundle("mysql.select.msg.all");
+	private static String sqldelete = GetConfig.getResourceBundle("mysql.delete.msg.all");
+	private DataSource ds;
```

Comment:
Codacy found an issue: [Perhaps 'ds' could be replaced by a local variable.](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=6170659)